sdk/scheduler/src/main/java/com/mesosphere/sdk/config/DefaultConfigurationUpdater.java

```diff
@@ -305,4 +310,18 @@ private void clearConfigsNotListed(Set<UUID> neededConfigs) throws ConfigStoreEx
             configStore.clear(configToClear);
         }
     }
+
+    private UpdateResult.DeploymentType getLastCompletedUpdateType() {
```

Comment:
Move to StateStoreUtils to be adjacent to e.g. setSuppressed/isSuppressed which is doing something similar